src/ansi-c/goto-conversion/goto_convert_class.h

```diff
@@ -56,6 +56,25 @@ class goto_convertt : public messaget
   std::string tmp_symbol_prefix;
   lifetimet lifetime = lifetimet::STATIC_GLOBAL;
 
+  struct clean_expr_resultt
```

Comment:
Can we add some documentation here and guidance on when to kill the temporaries in the presence of conditional jumps ?

Reply:
Done.